### PR TITLE
Exit with error message for missing kubeconfig instead of panic

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

File: pkg/clients/clients_test.go [hunk 1]
```diff
@@ -28,15 +28,19 @@ var _ = Describe("Client", func() {
 	When("A clientset is requested with no kubeconfig", func() {
 		It("should panic", func() {
```
Comment (crwr45) [inline pkg/clients/clients_test.go:29]: It should no longer panic, it should return an error.

File: pkg/cmd/root.go [hunk 2]
```diff
@@ -39,6 +39,7 @@ var (
 		Long:  `A monitoring tool for PTP related metrics.`,
 		Run: func(cmd *cobra.Command, args []string) {
 			logging.SetupLogging(logLevel, os.Stdout)
+
```
Comment (crwr45) [inline pkg/cmd/root.go:42]: I'd prefer not to have unrelated whitespace changes like this, it makes the change set less clear.

**Review by crwr45 — APPROVED** [review]
Minor tweaks, then it's good.

File: pkg/cmd/root.go (hunk 2 above)
Comment (nocturnalastro) [reply pkg/cmd/root.go:42]: Sorry about that
